rhcos4/profiles/moderate.profile

```diff
@@ -1,5 +1,11 @@
 documentation_complete: true
 
+metadata:
+    policy_hub: https://www.fedramp.gov/documents/
+    current_version: N/A
```

Comment:
Adding this doesn't make sense, it should be the default, and if there is not version, it is not applicable.

Reply:
I don't understand whats hould be the default.
I removed the `version` entry anyway.